modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -306,7 +306,7 @@ CV_ALWAYS_INLINE void absdiff_store(float out[], const v_float32& a, const v_flo
 template<typename T, typename VT>
 CV_ALWAYS_INLINE int absdiff_impl(const T in1[], const T in2[], T out[], int length)
 {
-    constexpr int nlanes = static_cast<int>(VT::nlanes);
+    const int nlanes = VT::nlanes;
```

Comment:
Is this `constexpr` to `const` change needed? If not, please restore it. Please revert also all similar cases below.

Reply:
It is if nlanes is determined at runtime

Reply:
Sure, but here we are not yet using a runtime `nlanes`, so it seems a bit premature to do this change.